Jenkinsfile

```diff
@@ -84,6 +84,7 @@ timestamps {
         robotest : {
           if (env.RUN_ROBOTEST == 'run') {
             withCredentials([
+                [$class: 'StringBinding', credentialsId: 'GET_GRAVITATIONAL_IO_APIKEY', variable: 'GET_GRAVITATIONAL_IO_APIKEY'],
```

Comment:
Still need this with https://github.com/gravitational/gravity/pull/2023?

Reply:
This is no longer needed because of #1997. #2023 isn't relevant to this PR because it targets 7.0.x and upgrade-via which hasn't hit master yet (as far as I know).

There will be a bit of merge headache between https://github.com/gravitational/gravity/issues/1915 and the upgrade via work, but nothing out of the ordinary given our port strategy.